### ISSUE TITLE
Login History Backend

### ISSUE THREAD

**Issue by crchong1** [keepid/keepid_server#12]
![image](https://user-images.githubusercontent.com/13875633/88863593-36718200-d1d1-11ea-9b5e-c5e5006e6026.png)

We want to indicate the login history of all users that can be seen in their access history under account settings. I think what would be the most helpful would be to indicate:

- Date
- IP Address (and we can pull the physical address from there, see below)
- maybe device (if that isn't too difficult, read here: https://stackoverflow.com/questions/8515161/detecting-device-type-in-a-web-application and here https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent)

I would sign up for https://ipinfo.io/ 
Here is the github https://github.com/ipinfo/java

it seems they have a free tier of up to 100k uses/month, which should suit us fine

Basically, when a login gets triggered, then store all this information in the db, maybe have an array in the user object that gets appended, with maybe a max length of like 1000 (to store the last 1000 logins).  

Then, add a POST route (maybe call it /get-login-history or something) which will then query the db for the user object and obtain the array. Once this is done, we can work on the front end.
